opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQParams.hpp

```diff
@@ -46,6 +47,22 @@ namespace Opm {
         std::mt19937& true_rng();
 
         bool operator==(const UDQParams& data) const;
+
+        template<class Serializer>
+        void serializeOp(Serializer& serializer)
+        {
+            serializer(reseed_rng);
+            serializer(random_seed);
+            serializer(value_range);
+            serializer(undefined_value);
+            serializer(cmp_eps);
+
+            auto now = std::chrono::high_resolution_clock::now();
+            auto ns = std::chrono::duration_cast<std::chrono::nanoseconds>(now.time_since_epoch());
+            this->m_true_rng.seed( ns.count() );
+            this->m_sim_rng.seed( this->random_seed );
+        }
```

Comment:
@joakim-hove needs to judge this.

Reply:
It works in the old serialization code, so it should work here as well.

Reply:
ref https://github.com/OPM/opm-common/blob/master/src/opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQParams.cpp#L80